### ISSUE TITLE
Issue of Issues

### ISSUE THREAD

**Issue by aarontitus** [CrisisCleanup/crisiscleanup-3-web#1012]
**Describe the bug**
These are all of the bugs and features that I have not yet fully documented, but need work:

Chrome 85.0.4183.102 64 bit, Windows 10.
Filters doesn't work.
New Cases "x" does not work.
Clear indicator that I'm on phone on every page.
Need a "Dismiss" and "Skip Call": Then tell me that I'll still get the call and need to press 0, and they won't be called.
On Hold Now is always 0.

Bug: https://www.crisiscleanup.org/incident/211/cases/new Table View is not working. 
Bug: Add general ticket for Chrome: No loading, and no filters. New users.

Bug: Hurricane Sally color is different in different browser tabs.
Bug: Defaults to Hanna. For Ariel and Reed.
Bug: @Braden Mars Reed reports that the first three calls successive calls this morning dropped after he pressed 1. The call would say it is connecting, but just never connect. The first successful connection occurred at about 10:30am Central.
He also gave us a clue about what might be happening with the Locked Phone Number problem (multiple calls to the same person). We know that you need to close out the phone call in Beta for it to unlock. However, Reed reports that sometimes the number in Beta is NOT the number that he's stuck on, and he needs click through two or three numbers in beta to close out the correct phone number to unlock.
Bug: @Tobi Abiodun Also, whenever he refreshes the page, he would go back to Hurricane Hanna, instead of Sally, even if Sally was the current incident. I'm pretty sure that user.states is not updating the default incident.
Feature: Add my Layers for "Layers", and for filters as well. Multi select is OR
Bug:  I've had several people say that they are getting calls from a 202 number, which I just wrote off until I realized that the AWS caller ID is actually a 202 number. And not our 800 number or even the 601 number. Is there any way we can fix that?
And incidentally, all 202 calls are blocked on my phone, so even after it went to VM and I did not press 1 to accept, I was still online. It should have gone offline.

Bug:We started having challenges with the search API yesterday but haven't been able to pin down the issue. For example, we noticed that a search for "Aaron Titus" will return me, but "Aaron Titu" will not. I suspect that issue has the same root cause. I'll put this on our to-do list.
You are the second person to ask about what the calls/ days means. It means "The survivor has called three times in the past two days," and is not an indication of the number of times we have called back.
We'll look into clarifying that meaning.

The issue with skipping is that the indication that a certain phone number is associated with a case # doesn't come up automatically anymore.
The work order number SOMETIMES populates after the person starts talking OR we have to add their name and part of an address before it is recognized.
As I recall, early on, the system USED to indicate the existence of a work order much sooner.

Another question - help us understand what the "3 calls 2 days ago"means. Is that the survivor called 3 times 2 days ago or is it that volunteers have attempted to call the survivor 3 times 2 days ago?

Bug: I am doing callbacks. Caller states they called 1 time. We called them back and created workorder. Now I am calling again. Somehow the number isn't getting deleted after we return the call. It has happened to me 4 times in 2 days, which isn't that much, but I feel like I'm bothering them. Is this what the "4 calls/ 3 days ago" means? Should I bypass a cal if it says that?
Bug: Reed: Inbound call, press 1 to accept. You pressed 1, 
On the line, then phone beeped. It was another crisis cleanup call coming in, asked to press 1. Pressed 1. Rather than doing anything, but instead of doing anything, it just disconnected and did not send a call.
The phone number on the screen is different each time. It brought up the popup.

Kicked me over to Hanna, defaulted to Hanna. V.3 does not like Hanna. It does strange things. It is incosistent in operation. If it'm inadverently in Hanna, then it will switch over. But it kicked over to Sally all by itself.

Bug: @Braden Mars I just got a call from Reed. He clicks "Skip Call," and the system immediately sends him another call. He clicks "Skip Call" again, and the system immediately sends him another call. He repeated this four times before the system placed him in an "Offline" state and the calls stopped coming.
Bug: Reed reports another bug with the recent changes. He answered a call. While he was on the call, the center of the screen went blank. E.g. the center of the screen went white, but the top of the screen and the navigation bar still appeared. He completed the phone call, but had to click the Work button in order to enter the case. He clicked the Work button, and then back to the Phone button. When he returned to the phone system, the call did not appear in his most recent call list. Immediately thereafter, the phone system called the same lady again.
Bug: Move Organizations does not work

Bug: "Charles Severs" returns result, but "Charles Sever" Does not
Bug: move icon does not work
Bug: Filter is not working for Chrome. EM guy Tomkins. HIstory does not work.
Bug: When an organization is not "Active," users can still log in and do everything. We should design what they see if the organization is not active.
Bug: no new or redeploy alerts since 9/12. In messages?
Feature: Add age of case
Feature: Add Crisis Cleanup release, plus the organizational release, which I will manually enter into all of the organizational profiles, and will print only in English.

Feature: change email, user initiated
Feature: Implement history tables for any table without invalidated_at
Feature: General stats: My organization
Feature: Add filters by date: Created at or updated_at. Slide through the time, and shows where the areas are most damaged.
Feature: Add a "Troubleshooting" section with "Reset User States" button and "Reset Settings and Favorites" in profile. This would allow users to click a button to manually destroy their `user_users.states` and `user_users.preferences` for troubleshooting purposes. Include description.
Feature: Add reports events to new table. Once complete, let me know and I will manually reconstruct all of the previous report views from email alerts
Feature: Invite to an affiliate or group organization 
Feature: Find case by team/ team lead
Feature: Find team by case
Feature: Add ghost users to team
Feature: Create multi-stake team
Feature: Allow any stake to approve orgs from other deployed stake to the same incident.
Feature: Multi stake command center
Feature: Allow create teams for any affiliated organization
Feature: Sort by date of claim and completion.
Feature: Hover on case to get case number
Feature: Add ghost_created_by, ghost_updated_by on worksite_worksites_users_time
Feature: Turn off reports emails
Bug: Phone numbers don't appear in top-right corner
Bug: Request access no work Samsung Chrome, can't select org

**Comment by BradenM** [comment]
> Bug: When I open crisiscleanup.org after having closed my browser and restarted my computer, I didn't need to log in again. That seems like a security flaw.

This is because your JWT token used for authentication is stored by your browser. The token does have an expiration, however. So if you were to shutdown your computer for a while and then come back, you would be logged out.

**Comment by BradenM** [comment]
> The work order number SOMETIMES populates after the person starts talking OR we have to add their name and part of an address before it is recognized.
As I recall, early on, the system USED to indicate the existence of a work order much sooner.

Is this in reference to phone? If so, is it still an issue? If also so, where and when is it taking a while to populate? (In popup or in controller?)
